src/store/modules/coaches/getters.js

```diff
@@ -4,5 +4,10 @@ export default {
   },
   hasCoaches(state) {
     return state.coaches && state.coaches.length > 0;
+  },
+  isCoach(_, getters, _2, rootGetters) {
+    const coaches = getters.coaches;
+    const userId = rootGetters.userId;
+    return coaches.some(coach => coach.id === userId)
   }
 };
```

Comment:
@HVRolim newline no final do arquivo
```suggestion
};

```